packages/cli/src/cli.test.ts

```diff
@@ -0,0 +1,275 @@
+import {
+    afterAll,
+    beforeAll,
+    beforeEach,
+    describe,
+    expect,
+    it,
+    vitest,
+} from 'vitest';
+import fs from 'fs';
+
+import { cli } from './cli';
+import { randomUUID } from 'crypto';
+import { assert } from 'console';
+import path from 'path';
+
+describe('CLI tests', () => {
+    describe('`generate` command tests', () => {
+        const url = 'postgres://postgres:postgres@localhost:5432/postgres';
+        const out = 'generated/synthql/tests';
+        const defaultSchema = 'pg_catalog';
+        const schemas = ['public'];
+
+        beforeAll(() => {
+            fs.rmSync(out, {
+                force: true,
+                recursive: true,
+            });
+        });
+
+        beforeEach(() => {
+            vitest.resetModules();
+            vitest.resetAllMocks();
+        });
+
+        it(
+            'Generate schema types with the CLI options default values',
+            async () => {
+                const outDir = `${out}/${Date.now()}-${randomUUID()}`;
```

Comment:
While this is totally fine, you could have just done randomUUID or even just Math.random(). The probability of a colission is super low.

Reply:
No need to change anything: just an informational message.